app/src/ui/branches/pull-request-list-item.tsx

```diff
@@ -49,6 +49,13 @@ export interface IPullRequestListItemProps {
   /** When a drag element has landed on a pull request */
   readonly onDropOntoPullRequest: (prNumber: number) => void
 
+  /**
+   * A callback which is invoked when the button's context menu
+   * is activated. The source event is passed along and can be
+   * used to prevent the default action or stop the event from bubbling.
+   */
+  readonly onContextMenu?: (event: React.MouseEvent<HTMLDivElement>) => void
```

Comment:
This approach unfortunately only works for mouse users. In order for the keyboard shortcuts to work, the context menu invocation must be on the div with a role of option. 

Fortunately, it isn't too difficult to fix. :) The `<SectionFilterList<T>` has a prop of `onItemContextMenu`. You can see an example of it implemented for the branch list in the `branch-list.tsx`